pkg/proxy/iptables/proxier.go

```diff
@@ -128,14 +131,25 @@ const sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
 
 // internal struct for string service information
 type serviceInfo struct {
-	clusterIP           net.IP
-	port                int
-	protocol            api.Protocol
-	nodePort            int
-	loadBalancerStatus  api.LoadBalancerStatus
-	sessionAffinityType api.ServiceAffinity
-	stickyMaxAgeSeconds int
-	externalIPs         []string
+	clusterIP              net.IP
+	port                   int
+	protocol               api.Protocol
+	nodePort               int
+	loadBalancerStatus     api.LoadBalancerStatus
+	sessionAffinityType    api.ServiceAffinity
+	stickyMaxAgeSeconds    int
+	externalIPs            []string
+	onlyNodeLocalEndpoints bool
+	healthCheckNodePort    int
+	guid                   types.UID
```

Comment:
As far as I can tell, most of the UIDs in these structs are for logging.  Do we really need them?  I don't mind them for logs so much but I dislike them in the iptables ocmments (which are already long)